tools/loadout_converter/loadout_converter.py

```diff
@@ -0,0 +1,68 @@
+# Converts all existing loadout `gear` listings in the `characters` table from their 'display_name's to their typepaths.
```

Comment:
This file most likely needs a few review passes. It all works fine in testing, but I haven't coded anything in Python for about 1-2 years now so there's bound to be mistakes.